stacktrace.js

```diff
@@ -163,12 +163,12 @@
         },
 
         /**
-         * Given an Array of StackFrames, serialize and POST to given URL.
+         * Given an error message and Array of StackFrames, serialize and POST to given URL.
          *
          * @param {Array} stackframes
          * @param {String} url
          */
-        report: function StackTrace$$report(stackframes, url) {
+        report: function StackTrace$$report(errorMsg, stackframes, url) {
```

Comment:
Are you suggesting to move it to last?  Or is creating a completely separate method better?